### ISSUE TITLE
Upgrade ArcGIS Enterprise from 10.7. to 10.7.1 - DataStore Upgrade failed silently

### ISSUE THREAD

**Issue by Biboba** [Esri/arcgis-powershell-dsc#188]
I just upgraded from 10.7 to 10.7.1 and though everything seemed to be successful, the Hosting ArcGIS Server failed to validate my tileCache datastore. 
Having a look at the log, I noticed that DataStoreConfigUpgrade failed silently:

```
[STORE02]: LCM:  [ Start  Set      ]
[STORE02]: LCM:  [ Start  Resource ]  [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade]
[STORE02]: LCM:  [ Start  Test     ]  [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade]
[STORE02]: LCM:  [ End    Test     ]  [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade]  in 36.1710 seconds.
[STORE02]: LCM:  [ Start  Set      ]  [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade]
[STORE02]:                            [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade] Retrieved server token successfully
[STORE02]: LCM:  [ End    Set      ]  [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade]  in 8.9060 seconds.
[STORE02]: LCM:  [ End    Resource ]  [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade]
[STORE02]: LCM:  [ End    Set      ]
[STORE02]: LCM:  [ End    Set      ]    in  45.3590 seconds.
Operation 'Invoke CimMethod' complete.
````

I had to finish the job manually from DataStore web interface and it was successful.

Thanks

**Comment by shailesh91** [comment]
@Biboba it would be great if you could please attach JSON config files. It helps us immensely to repro the issue.

**Comment by Biboba** [comment]
Here you are:
[ProdDeploymentUpgrade.txt](https://github.com/Esri/arcgis-powershell-dsc/files/3373750/ProdDeploymentUpgrade.txt)

Thanks

**Comment by shailesh91** [comment]
@Biboba I noticed you mentioned that only tile cache validation failed, was your relational datastore able to validate?

**Comment by Biboba** [comment]
@shailesh91, Yes I was able to validate my relational datastore. There was no issue with it.

**Comment by shailesh91** [comment]
@Biboba  I have provided some stability fixes in V2.1.1, that should try to overcome this issue. Please let me know if v2.1.1 works for you.

**Comment by shailesh91** [comment]
@Biboba I am closing this issue for now due inactivity. If you face this issue again please reopen.
